include/api.php

```diff
@@ -130,6 +130,30 @@ function pll__( $string ) {
 }
 
 /**
+ * Translates a string ( previously registered with pll_register_string ) and escapes it for safe use in HTML output.
+ * 
+ * @since 2.1
+ * 
+ * @param string $string the string to translate
+ * @return string translation in the current language
+ */
+function pll_esc_htlm__ ( $string ) {
```

Comment:
There is a space between the function name and the opening bracket. HTML has still a typo.